security-analysis/security-analysis-api/src/test/java/com/powsybl/security/SecurityTest.java

```diff
@@ -130,6 +131,16 @@ void checkLimits() {
         assertViolations(violations);
     }
 
+    /*@Test
+    void checkLimitsWithThreeWindingsTransformer() {
```

Comment:
In the factory, you must add values in the three terminals of the three windings transformer. 
````
        ThreeWindingsTransformer twt = network.getThreeWindingsTransformer("toto");
        twt.getTerminal(ThreeWindingsTransformer.Side.ONE).setP(1400);
        twt.getTerminal(ThreeWindingsTransformer.Side.ONE).setQ(400);
````

Reply:
done